<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InAppDialog</class>
 <widget class="QDialog" name="InAppDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="TwoLineButton" name="monthlySubscriptionButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>461</width>
     <height>62</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: 1px solid #0F7267; border-radius: 4px;} 
QPushButton:hover { background-color:#1A7F67; }
QPushButton:pressed {background-color:#0A5A50; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="TwoLineButton" name="triMonthSubscriptionButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>461</width>
     <height>62</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: 1px solid #0F7267; border-radius: 4px;} 
QPushButton:hover { background-color:#1A7F67; }
QPushButton:pressed {background-color:#0A5A50; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="TwoLineButton" name="yearlySubscriptionButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>461</width>
     <height>62</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: 1px solid #0F7267; border-radius: 4px;} 
QPushButton:hover { background-color:#1A7F67; }
QPushButton:pressed {background-color:#0A5A50; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="subscriptionTipsTextEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Unlock MovieMator Video Editor&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To save videos with no watermark &amp;amp; no duration limitation and get the best video editing experience.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="termOfUseButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline; </string>
   </property>
   <property name="text">
    <string>Term of Use</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="privacyPolicyButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline; </string>
   </property>
   <property name="text">
    <string>Privacy Policy</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="restorePurchaseButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline; </string>
   </property>
   <property name="text">
    <string>Restore Purchase</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="paymentTipsTextEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>461</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Subscriptions will be charged to your iTunes account as soon as the purchase is confirmed. Subscriptions are billed monthly, once every 3 month or annually at the rate selected depending on the subscription plan. Subscriptions will renew automatically within 24 hours of the end of the current subscription period. Subscriptions will renew automatically unless auto-renew is turned off at lease 24 hours prior to the end of the current billing period. You may manage your subscription and turn off auto-renewal by going to your iTunes Account Settings.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="TwoLineButton" name="useFreeVersionButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>461</width>
     <height>62</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: 1px solid #0F7267; border-radius: 4px;} 
QPushButton:hover { background-color:#1A7F67; }
QPushButton:pressed {background-color:#0A5A50; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TwoLineButton</class>
   <extends>QPushButton</extends>
   <header>widgets/twolinebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
